src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -713,7 +713,7 @@ else if (targetFile.exists())
         {
             Files.copy(file.toPath(), targetFile.toPath());
         }
-        catch (IOException e)
+        catch (final IOException e)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1166)